tests/base_real_sensor_mock_test_case.py

```diff
@@ -88,7 +90,7 @@ def read(self, length: int) -> str:
 
         Returns
         -------
-        `str`
+        `str` or `bytes`
```

Comment:
I gently suggest `|` instead of `or` -- only because it is the standard way to indicate "or" for types. "or" is arguably more readable.